docs/reference/how-to/recipes.asciidoc

```diff
@@ -246,18 +246,18 @@ And these index statistics may be different across copies of the same shard
 due to deleted documents. As you may know when documents are deleted or updated,
 the old document is not immediately removed from the index, it is just marked
 as deleted and it will only be removed from disk on the next time that the
-segment this old document belongs to is merged. However for practical reasons,
+segment that this old document belongs to is merged. However for practical reasons,
 those deleted documents are taken into account for index statistics. So imagine
 that the primary shard just finished a large merge that removed lots of deleted
 documents, then it might have index statistics that are sufficiently different
-from the replica (which still have plenty of deleted documents) so that scores
-are different too.
+from the replica (which still have some documents as the sync has not completed
```

Comment:
This is somewhere between misleading and wrong, I think. There isn't a process that syncs merges from primary to replica - the replica just does the merge as well.